src/page-response-impact/BaselinePopulationModal/BaselinePopulationForm.tsx

```diff
@@ -0,0 +1,175 @@
+import numeral from "numeral";
+import React, { useState } from "react";
+import styled from "styled-components";
+
+import Colors from "../../design-system/Colors";
+import InputButton from "../../design-system/InputButton";
+import InputDate from "../../design-system/InputDate";
+import InputTextNumeric from "../../design-system/InputTextNumeric";
+import { Props as ModalProps } from ".";
+
+const ModalFooter = styled.div`
+  display: flex;
+  flex-flow: row nowrap;
+  justify-content: space-between;
+  padding: 1em 0;
+  width: 100%;
+`;
+
+const FormRow = styled.div`
+  align-items: center;
+  display: flex;
+  flex-flow: row nowrap;
+  justify-content: space-between;
+  margin: 4vw 0;
+`;
+
+const LabelContainer = styled.div`
+  display: flex;
+  flex-flow: column;
+  line-height: 150%;
+  letter-spacing: 0.15em;
+`;
+
+const Label = styled.div`
+  font-family: "Poppins", sans serif;
+  font-weight: normal;
+  font-size: 10px;
+  line-height: 150%;
+  text-transform: uppercase;
+  color: ${Colors.darkForest};
+`;
+
+const SubLabel = styled.div`
+  font-family: "Helvetica Neue", sans serif;
+  font-weight: 300;
+  font-size: 14px;
+  color: ${Colors.opacityForest};
+`;
+
+const Text = styled.div`
+  color: ${Colors.opacityForest};
+  font-family: "Poppins", sans serif;
+  font-size: 12px;
+  font-weight: normal;
+  line-height: 150%;
+  padding-top: 20px;
+`;
+
+const InputContainer = styled.div`
+  width: 170px;
+`;
+
+const FormContainer = styled.div`
+  padding: 2vw;
+`;
+
+const buttonStyle = {
+  width: "200px",
+  fontFamily: "PingFang SC",
+  fontSize: "14px",
+};
+
+type Props = Omit<ModalProps, "open" | "numFacilities"> & {
+  setPage: () => void;
+};
+
+const BaselinePopulationForm: React.FC<Props> = ({
+  defaultIncarceratedPopulation,
+  defaultStaffPopulation,
+  saveBaselinePopulations,
+  setPage,
+}) => {
+  const defaultTotalPopulation = numeral(
+    defaultIncarceratedPopulation + defaultStaffPopulation,
+  ).format("0,0");
+
+  // Default Original Date 3/1/2020
+  const defaultDate = new Date(2020, 2, 1);
+
+  const [populations, setPopulations] = useState({
+    staffPopulation: defaultStaffPopulation,
+    incarceratedPopulation: defaultIncarceratedPopulation,
+    date: defaultDate,
+  });
+
+  function handleInputChange(property: string, value: number | Date) {
+    setPopulations({ ...populations, [property]: value });
+  }
+
+  function handleSubmit() {
+    saveBaselinePopulations(populations);
+  }
+
+  return (
+    <>
+      <Text>
+        If you skip this step, we will assume a starting incarcerated population
+        of {defaultTotalPopulation}, which is derived from Vera data, and your
```

Comment:
I wonder if it would be more precise/formal to say "which is derived from data from the Vera Institute of Justice"?

Reply:
Assuming that copy is from @sychang, Serena have we vetted that with Clem or Andrew? I want to make sure we attribute things in the right way.

Reply:
Checked with Andrew, let's make it more generic to be "based on public data" for a couple reasons
1. If we change the source of data we don't have to change the copy
2. We want to avoid spotlighting any advocacy-focused groups directly in the tool

Reply:
Checked, let's make it more generic to be "based on public data" so if we change the source of data we don't have to change the copy.